build/secondary/testing/gtest/BUILD.gn

```diff
@@ -18,7 +18,10 @@ config("gtest_config") {
   ]
 
   # Gtest headers need to be able to find themselves.
-  include_dirs = [ "include" ]
+  include_dirs = [
```

Comment:
> That would mean looking at "//build/secondary/testing/gtest/include". There is no such path. The gtest checked out by deps is in //testing/gtest

I don't think that's how build/secondary works - in GN paths inside the secondary directory should map to their non-secondary locations transparently.  Do you have a 'gn check' failure showing the tool looking in the wrong place?